.github/workflows/bare.yaml

```diff
@@ -36,7 +34,7 @@ jobs:
       # Workflow won't be cancelled at the first failed job.
       fail-fast: false
       matrix:
-        runs-on: [ windows-2022, windows-2019 ]
+        runs-on: [ 'windows-2022' ]
```

Comment:
maybe remove the matrix from here

Reply:
Done through https://github.com/chevah/pythia/pull/39/commits/7d5e2275b13f021120c2e4a4e5687b0805707f16.